<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ErrorWindow</class>
 <widget class="QWidget" name="ErrorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>306</width>
    <height>175</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextBrowser" name="ErrorBrowser">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>301</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ExitButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>140</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Закрыть</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
